src/node_quic.cc

```diff
@@ -251,6 +253,46 @@ int Server_Transport_Params_Parse_CB(
   return 1;
 }
 
+void InitKeylog(Environment* env, SecureContext* sc) {
+  static constexpr const char* LF = "\n";
```

Comment:
```suggestion
  static const char LF = '\n';
```

+ suggestion below, or, if you don’t like that:

```suggestion
  static const char[] LF = "\n";
```